src/replica/replication_app_base.h

```diff
@@ -123,6 +126,8 @@ class replication_app_base : public replica_base
     typedef replication_app_base *factory(replica *r);
     static void register_storage_engine(const std::string &name, factory f);
     static replication_app_base *new_storage_instance(const std::string &name, replica *r);
+    static const std::string kDataDirPostfix;
+    static const std::string kRdbPostfix;
```

Comment:
Need to also replace `"data"` and `"rdb"` in other files such as `src/server/pegasus_server_impl.cpp` `src/server/test/pegasus_server_test_base.h` ?